src/main/scala/uncore/converters/Tilelink.scala

```diff
@@ -278,11 +282,22 @@ class TileLinkIOWidener(innerTLId: String, outerTLId: String)
     data = put_data.asUInt,
     wmask = Some(put_wmask.asUInt))(outerConfig)
 
+  val put_atomic_acquire = PutAtomic(
+    client_xact_id = iacq.client_xact_id,
+    addr_block = out_addr_block,
+    addr_beat = out_addr_beat,
+    addr_byte = out_addr_byte,
+    atomic_opcode = iacq.op_code(),
+    operand_size = iacq.op_size(),
+    data = align_data(switch_addr, iacq.data))(outerConfig)
+
   io.out.acquire.valid := sending_put || (!shrink && io.in.acquire.valid)
-  io.out.acquire.bits := MuxCase(get_block_acquire, Seq(
+  io.out.acquire.bits := MuxCase(Wire(io.out.acquire.bits, init=iacq), Seq(
```

Comment:
Is there some way to have the default be an assertion or something? It seems like that would have made this problem easier to find.